domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/base/ErrorNotice.java

```diff
@@ -35,6 +35,7 @@
     protected static final String E_015 = "E015";
     protected static final String E_016 = "E016";
     protected static final String E_017 = "E017";
+    protected static final String E_018 = "E018";
```

Comment:
I would strongly suggest that we start documenting each of these rules in a human readable markdown file like https://github.com/CUTR-at-USF/gtfs-realtime-validator/blob/master/RULES.md along with each of these PRs.

From my experience with the RT validator, it's far easier to document these as we go, as opposed to trying to go back later and re-read the code to remember what we implemented.

Reply:
Opening an issue on this matter